feature_server/commands.py

```diff
@@ -197,10 +213,16 @@ def say(connection, *arg):
     connection.protocol.send_chat(value)
     connection.protocol.irc_say(value)
 
-@admin
-def kill(connection, value):
+add_rights('kill', 'admin')
+def kill(connection, value = None):
+    if value is None:
+        player = connection
+    else:
+        if not connection.rights.kill:
+            return "You can't use this command"
     player = get_player(connection.protocol, value, False)
     player.kill()
+    if connection is not player:
```

Comment:
Indentation errors here. I'm guessing the next 2 lines should be indented and thus controlled by this `if`, but I'm not entirely sure what logic is wanted here.